django_freeradius/tests/test_api.py

```diff
@@ -52,3 +54,140 @@ def test_postauth_400(self):
         response = self.client.post(reverse('freeradius:postauth'), {})
         self.assertEqual(RadiusPostAuth.objects.all().count(), 0)
         self.assertEqual(response.status_code, 400)
+
+    def test_accounting_start(self):
+        self.assertEqual(RadiusAccounting.objects.all().count(), 0)
+        original = RadiusAccounting.objects.create(
```

Comment:
We don't need this for start
Actually we could have two tests, both with an existing accounting with the same id and one without it